dispatcher/stateful.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+module.exports = function StatefulDispatcher(dispatcher, data, onData) {
```

Comment:
Don’t copy and paste ;-) This is the *stateful* dispatcher.

Would you add a comment in the source or just document it?